src/main/java/org/opentripplanner/middleware/models/AbstractUser.java

```diff
@@ -60,6 +61,10 @@ public boolean canBeManagedBy(Auth0UserProfile user) {
 
     @Override
     public boolean delete() {
+        if (Auth0Connection.isAuthDisabled()) {
+            LOG.warn("Skipping delete user call (Auth is disabled).");
```

Comment:
Change the text to "Skipping call to delete Auth0 user (Auth is disabled)"

I would tweak this check as follows because the if condition only applies to deleting the Auth0 user:
```java
if (Auth0Connection.isAuthDisabled()) {
  ...
} else {
  try {
    deleteAuth0User(...)
  } catch (...) {
    ...
  }
}
```